### ISSUE TITLE
I Came Across George Floyd’s Identical Twin at a KFC, and Spawned a Floydzilla

### ISSUE THREAD

**Issue by celeronishe** [WinampDesktop/winamp#1248]
In light of some recent events, I have decided to share a story that I have had with one of the Floyd twins. It’s kind of a complicated story so I have to give you some context first.
Many of you have probably heard of George Floyd before. He was a monkey who died because of a fentanyl overdose. However, what many people do not know is that the Floyd family had 2 kids who were twins. They were both named George and are exactly identical, so there is actually 2 George Floyds in the world. Oh, and they also have the exact same interests. Alright, now onto the story.
One day, I was feeling hungry, and my fridge was empty. I didn’t feel like going to the supermarket, so I decided to get some fast food. While I was driving and listening to the radio, I heard a commercial for a new KFC meal. It sounded like it would be tasty as fuck, so I decided I to go there. When I got there, the drive-thru line was like 5 billion cars long. There was no fucking way I was going to wait in that shit, so I decided to order carryout. I walked inside and then ordered my food. While I was waiting on my phone, I looked up and I noticed that a black man entered the building. 
“Oh great, another nigga in the KFC,” I thought to myself. 
After a few minutes, my food was ready and soon enough, I was on my way to walk out of the building. But then, I was suddenly stopped by the same nigga who I noticed had entered the KFC earlier. 
“OOGA BOOGA. GIVE ME KFC!” it said. 
I said fuck off nigga and started sprinting towards my car. But of course, I was running against a black man, so I was going to get outrun. He then snatched the KFC out of my hands and consumed all of it within a few seconds. I have no fucking clue how he did it, but afterward, he started choking. But he soon got it all down his stomach and said “Phew, I couldn’t breathe for a second there.” 
I was pissed off as anybody would be in that situation, so I decided to go to Subway and get my usual sandwich. Thankfully, no monkeys came and stole my food this time, so I was finally able to have a nice peaceful lunch. I then went back home and relaxed for a few hours, watching YouTube videos and scrolling through Instagram as I usually do. 
But then somebody knocked on my door. I opened the door to see who it was. It was George Floyd. He apologized for stealing my KFC and asked if he could come into my house so we could chill for a little bit. I said no fuck off, but then the police came and told me he was an illegal immigrant and was going to be deported to Africa the next day, but he needed a place to stay for the night. And of course, that nigga chose my house. I was forced to let him into my house. Right when he walked in, the first thing he saw was my precious $2000 blender that I had gotten for Christmas the previous year. It was one of the best blenders in the world and could create breathtaking slushies and smoothies. 
He picks up a pair of bananas I have on my counter then goes looking through my pantry. At first, I had no idea what he was looking for but then I realized when he picked it up. Fentanyl. He put the fentanyl and bananas into my blender and started working it. My blender cannot blend drugs and if you try to do so, it will break. Well, guess what, that nigga broke my blender. I wanted to kick that monkey out so badly. But I couldn’t because he had to spend the whole night here. 
After he did that, he walked over to my cupboard full of VHS tapes. He picked up one of them, the one he picked up was the VHS of Clarissa Explains it All: Dating. If you don’t know what Clarissa Explains It all was, it was a nickelodeon sitcom in the early 90s.
“This girl looks hot. I want her pregnant belly pics,” George said. 
I do have to admit she was hot as fuck and that if I were in that show, I would definitely bang her. But I had to explain to him that for one it was a sitcom from the 90s and that the girl looks completely different now. And two, she is not pregnant. 
“OOGA BOOGA. NIGGA MAD!” he said. 
He then took all the fentanyl in my pantry and consumed all of it at once. He then started transforming into a giant creature. And of course, it went through my roof which means this nigga severely damaged my house. Then, he turned into his final form: Floydzilla. 
“I WANT FENTANYL AND PREGNANT BELLY PICS!” he roared. I ran away from him as fast as I could, and also that nigga stepped on my house which means now it is completely destroyed. My anger was unreal at the time. But then a miracle happened. 
I found 5 cups full of fentanyl. I consumed it all at once and then I also transformed into a giant. I then fought with Floydzilla for a solid 10 minutes. Once he was weakened, I slammed him and then started kneeling on his neck. 
“I CAN’T BREATHE!” he roared. 
After 8 minutes of kneeling on his neck, I killed him. I then started transforming back to normal. Apparently, George Floyd’s twin was a serial criminal throughout the United States. I was given a reward of $50 million by the government. With that money, I bought a mansion and a nice sports car and restarted my VHS collection. Since they are so cheap, I managed to get it the same size as my old collection within days. I got very lucky with this event, because most people would have gotten fed by George Floyd and received a bullet wound in their stomach due to them not having pregnant belly pics. 
In conclusion, be careful about where you eat. You never know what people may change your life for good or worse. And remember, fuck niggas as they contribute absolutely nothing to society.

**Comment by pelaajahacks** [comment]
hate when this happens. totally get why you did that
